airflow/exceptions.py

```diff
@@ -424,3 +424,11 @@ def __str__(self):
             "Error deserializing result. Note that result deserialization "
             "is not supported across major Python versions. Cause: " + str(self.__cause__)
         )
+
+
+class StopDagTest(AirflowException):
```

Comment:
I don’t think there’s anything to gain inheriting from AirflowException. It might be better to just make this inherit from Exception and put it in `dag.py` instead.